source/layout.erb

```diff
@@ -68,5 +69,6 @@
         var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
       })();
     </script>
+    <script src="https://glimmer-styleguide.global.ssl.fastly.net/glimmer-styleguide/master/app.js"></script>
```

Comment:
Should we use a simple helper for all these urls so it's easier to adjust when we inevitably need to in the future?